### ISSUE TITLE
Make Go more accessible to color-blind users

### ISSUE THREAD

**Issue by arvindsv** [gocd/gocd#2]
Here's feedback from a user of Go, who is color-blind:

> I am color blind and I find the Go UI very hard to use. It might be pretty obvious to use green and red as indicators on the Go UI, but at times I find it a little hard to differentiate between the two and this makes the Go UI extremely unusable to me. I can almost never tell if a stage has failed unless I click into the stage. If I stare hard enough, I can figure out, but it's not as obvious to me as it is to you. 
> 
> The simple fix for this is to use labels instead of color for build statuses. Though I understand that color blindness is a relatively uncommon condition (not very uncommon though, around 10% of all men are red-green color blind), if you do have this condition, you can imagine the GO UI as a bunch of grey lines with no indication of where the failures are.

Needs discussion. Should this be an option? Should we change something more basic?

**Comment by grahamc** [comment]
A small amount of CSS might fix this:

```css
.Passed:after {content: "✓";}

.Failed:after {content: "✗";}
  
.stage_bar {
  color: white; padding-left: 5px;
}
```

![pipelines_-_go](https://cloud.githubusercontent.com/assets/76716/9287205/b2573004-42cf-11e5-906d-1985d19d5f6f.jpg)

protanopia:
![simulate](https://cloud.githubusercontent.com/assets/76716/9287206/c2e44434-42cf-11e5-8c29-d26071fe2c1e.jpg)

deuteranopia:
![simulate-2](https://cloud.githubusercontent.com/assets/76716/9287220/e8f93440-42cf-11e5-823f-2958a530e629.jpg)

tritanopia:
![simulate-4](https://cloud.githubusercontent.com/assets/76716/9287221/fb47d610-42cf-11e5-9e45-252771eef094.jpg)

**Comment by ibnc** [comment]
Can this issue be closed now since #3233 was merged?

**Comment by arvindsv** [comment]
I think so. Closing this.
